keystrokes/src/components/Character/CharacterDetails/KeybindEditor.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
```

Comment:
I think i'd personally import these individually but that'd be kinda a premature optimization.

Reply:
i didnt realize there was a difference, wouldnt it just be abunch of `import { x } from y`?